wsmaster/codenvy-ldap/src/main/java/com/codenvy/ldap/sync/ProfileMapper.java

```diff
@@ -44,6 +44,7 @@ public ProfileMapper(String idAttr, Pair<String, String>[] attributes) {
             this.appToLdapAttrNames = ImmutableMap.of();
         } else {
             this.appToLdapAttrNames = ImmutableMap.copyOf(Arrays.stream(attributes)
+                                                                .filter(p -> !p.first.isEmpty() && p.second != null)
```

Comment:
AFAIK you can't get NULL injected where array of string pairs is injected, i think this check 
is useless and shouldn't be present here.